src/Microsoft.AspNetCore.TestHost/ClientHandler.cs

```diff
@@ -13,6 +13,7 @@
 using Microsoft.AspNetCore.Hosting.Server;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Http.Features;
+using Microsoft.Net.Http.Headers;
```

Comment:
Why was this needed?

Reply:
@Tratcher Ooops. 

It's a ghost of the shadow of the remnant of a previous code attempt.

The 👻 has just been busted by a team of professionals.

I've been told it shouldn't haunt us back.